setup.py

```diff
@@ -89,6 +89,7 @@ def read(f):
     "pytest-timeout",
```

Comment:
@webknjaz  If `aiohttp` already has tests that use `trustme` why isn't it listed as ` tests_require`?

Reply:
`tests_require` is an obsolete setuptools concept and should be eliminated.

Reply:
In fact, it's best to put test deps in requirements files — it's their use-case to describe venvs (like the one for testing)

Reply:
I'll put it in there, but should I just remove this section altogether?

Reply:
Looks like you commented here and deleted those comments. Let's not do any section removals or packaging refactoring in this PR.